installer/win/install_instructions_win.md

```diff
@@ -219,15 +219,15 @@ Expected values:
 
 #### Example, install with dev tools enabled
 
-    msiexec /i Symphony.msi DEV_TOOLS_ENABLED="true"
+    msiexec /i Symphony.msi DEV_TOOLS_ENABLED=true
```

Comment:
I think that since the quotation marks on the command line gets stripped out, we should revert to specifying them in the documentation ( so "true" rather than true) just for consistency with other parameters like paths which might need them.  We've had some confusion in the path over what needs quotation marks and what doesn't, so even if not strictly necessary, I think it will make our support slightly easier to specify them for everything

Reply:
Totally agree with you. I’m reverting this part.